samples/word/04-range/split-words-of-first-paragraph.yaml

```diff
@@ -0,0 +1,161 @@
+id: word-basics-split-words-of-first-paragraph-
+name: Range manipulation
+description: 'Show how to split a paragraph into word ranges and then traverses all the ranges to format each word, producing a "karaoke" effect.'
+author: Microsoft
+host: WORD
+api_set:
+    WordApi: 1.1
+script:
+    content: |-
+        $("#setup").click(setup);
+        $("#highlight").click(highlightWords);
+
+        function highlightWords() {
```

Comment:
ok, for now i will keep it like this. i will typescript it later if time permits.